<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">RePKG-GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/ico/repkg_gui.png</normaloff>:/ico/repkg_gui.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
  border-radius: 3px;
  border: 1px solid grey
}

QPushButton:focus {
  border: 1px solid #0078d4
}

QLineEdit {
  border-radius: 3px;
  border: 1px solid grey
}

QLineEdit:focus {
  border: 1px solid #0078d4
}

QPlainTextEdit {
  border: 1px solid grey
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabExtract">
     <attribute name="title">
      <string>Extract</string>
     </attribute>
     <widget class="QPushButton" name="pb_selectSourcePath">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Browse</string>
      </property>
      <property name="text">
       <string notr="true">...</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="DropableLineEdit" name="dle_sourcePath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Drop .pkg file here</string>
      </property>
     </widget>
     <widget class="DropableLineEdit" name="dle_outputPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Output directory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_selectOutputPath">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Browse</string>
      </property>
      <property name="text">
       <string notr="true">...</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Extract">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>490</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Extract</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_dontConvertTex">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Don't convert TEX into images</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ExtractResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInfo">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QLabel" name="label_notSupported">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>441</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sorry, this is currently not supported.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSettings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_defaultOutputPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Default output path (regex)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dle_defaultOpenPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>441</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>If empty, will use the path from the last time</string>
      </property>
      <property name="placeholderText">
       <string>If empty, will use the path from the last time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_defaultOpenPath">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Browse</string>
      </property>
      <property name="text">
       <string notr="true">...</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_autoExec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Execute on open/drop</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_ar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto find Steam workshop dir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dle_defaultOutputPathMatch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>471</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Match rule</string>
      </property>
      <property name="placeholderText">
       <string>Match rule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dle_defaultOutputPathReplace">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>471</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Replace rule</string>
      </property>
      <property name="placeholderText">
       <string>Replace pattern</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_defaultOpenPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Default open path</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QPlainTextEdit" name="pte_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbout">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label_repkgVersionLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RePKG Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_repkgVersion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>471</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: grey</string>
      </property>
      <property name="text">
       <string notr="true">Unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_webSites">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>450</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://www.qt.io&quot;&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline; color:#1f9b5d;&quot;&gt;Qt Project&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://github.com/Tianming-Wu/RePKG-GUI&quot;&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline; color:#1f9b5d;&quot;&gt;RePKG-GUI&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://github.com/notscuffed/repkg&quot;&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline; color:#1f9b5d;&quot;&gt;RePKG&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_author">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Bilibili	&lt;a href=&quot;https://space.bilibili.com/671411793&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#1f9b5d;&quot;&gt;@不知干啥的天明同学&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Github	&lt;a href=&quot;https://github.com/Tianming-Wu&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#1f9b5d;&quot;&gt;Tianming Wu&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::RichText</enum>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>dropablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
